src/libs/actions/User.js

```diff
@@ -70,6 +71,8 @@ function closeAccount(message) {
             ],
         },
     );
+    // Clear all the listeners/callback that are registered while logged-in
```

Comment:
Little nitpick here; I think the comment could explain why we need to do this cleanup - otherwise it won't make sense for future readers of the code and we could just leave out the comment altogether.

Reply:
So shall I remove the comment? or update the comment?

Reply:
<img width="1037" alt="Screenshot 2023-08-30 at 6 40 15 PM" src="https://github.com/Expensify/App/assets/59088937/bdd9b63f-ba40-41b8-a08d-0056995ef47c">

Are these comment good enough?!

Reply:
Sure, sorry I'm a bit weak in commenting :)

Reply:
<img width="1007" alt="Screenshot 2023-08-30 at 6 54 17PM" src="https://github.com/Expensify/App/assets/59088937/10b08bc8-d6ce-4a0f-babb-a96b7f747fab">

@jjcoffee This works right?